homeassistant/components/steam_online/sensor.py

```diff
@@ -140,10 +147,28 @@ def _get_current_game(self):
         _LOGGER.error("Unable to find name of app with ID=%s", game_id)
         return repr(game_id)
 
+    def _get_last_online(self):
+        last_online = datetime.fromtimestamp(
+            mktime(self._profile.last_online)
+        ).isoformat()
+
+        if last_online:
+            return last_online
+
+    def _get_level(self):
+        level = self._profile.level
+
+        if level:
+            return level
+
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
-        return {"game": self._game} if self._game else None
+        return (
+            {"game": self._game, "last_online": self._last_online, "level": self._level}
+            if self._game or self._last_online or self._level
```

Comment:
Should we really make the total return value coupled to all attributes like this? Shouldn't we check each attribute and only set each attribute if it's not `None` independently of the other attributes?

Reply:
Split it into several lines with one if per value